docs/en/reference/sql/dql/WINDOW_CLAUSE.md

```diff
@@ -245,24 +252,24 @@ WINDOW w1 AS (PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING A
 
 ### Window Frame Max Size
 
-OpenMLDB在定义了元素，来限定窗口内条数。具体来说，可以在窗口定义里使用**MAXSIZE**关键字，来限制window内允许的有效窗口内最大数据条数。
-
+The keyword `MAXSIZE` is used to limit the number of rows in the window.
 ```sql
 WindowFrameMaxSize
 				:: = MAXSIZE NumLiteral
 ```
 
 ![Figure 6: window config max size](../dql/images/window_max_size.png)
 
-#### **Example: ROW RANGE 窗口MAXSIZE**
+####Example
+**1. ROWS RANGE WINDOW with MAXSIZE**
 
+The following `WINDOW` clause defines a `ROWS_RANGE` window containing preceding 10s rows and current row. There are at most 3 rows in the window.
 ```sql
--- ROWS example
 -- desc: window ROWS_RANGE, 前10s到当前条，同时限制窗口条数不超过3条
 SELECT sum(col2) OVER w1 as w1_col2_sum FROM t1
 WINDOW w1 AS (PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW MAXSIZE 3);
 ```
 
 ```{seealso}
-Aggregate functions that can be used in window calculation, refer to [Built-in Functions](../functions_and_operators/Files/udfs_8h.md)
+Aggregate functions that can be used in window computation, refer to [Built-in Functions](../functions_and_operators/Files/udfs_8h.md)
```

Comment:
Please refer to [Built-in Functions](../functions_and_operators/Files/udfs_8h.md) for aggregate functions that can be used in window computation.